apps/epic-web/src/components/app/layout.tsx

```diff
@@ -68,6 +71,7 @@ const Layout: FunctionComponent<React.PropsWithChildren<LayoutProps>> = ({
         noindex={noIndex}
       />
       <GlobalSearchBar />
+      <RedeemDialogForCoupon />
```

Comment:
So much nicer having this in the layout now instead of needing to embed it on each page where we might want to display the dialog.